portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase

```diff
@@ -889,4 +889,144 @@ definition {
+				title = "Web Content Title 2");
+		}
+
+		task ("Add a dynamic Collections and select Web Content Article as Asset Entry Type") {
+			JSONAssetlist.addDynamicAssetListEntryWithItemType(
+				groupName = "Test Site Name",
+				itemSubtype = "Basic Web Content",
+				itemType = "Web Content Article",
+				title = "Dynamic Collection");
+		}
+
+		task ("Set up the Personalized Variation") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection");
+
+			AssetListsAdmin.addPersonalizedVariation(
+				itemSubtype = "Basic Web Content",
+				itemType = "Web Content Article",
+				segmentName = "Segment Name");
+
+			AssetListsAdmin.addTagsFilter(tagNameList = "tag name");
+		}
+
+		task ("View all web content shown in collection items") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.viewContent(
+				assetListTitle = "Dynamic Collection",
+				assetTitle = "Web Content Title 1,Web Content Title 2",
+				assetType = "Web Content Article");
+		}
+
+		task ("View only the second web content shown in collection items") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.viewContent(
+				assetListTitle = "Dynamic Collection",
+				assetTitle = "Web Content Title 2",
+				assetType = "Web Content Article",
+				personalizationTitle = "Segment Name");
+		}
+
+		task ("Set up the Collection Display Fragment") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Content Display",
+				fragmentName = "Collection Display");
+
+			PageEditor.editCollectionDisplay(
+				assetListName = "Dynamic Collection",
+				fragmentName = "Collection Display");
+
+			PageEditor.addFragmentToCollectionDisplay(
+				collectionName = "Basic Components",
+				entryTitle = "Web Content Title 2",
+				fragmentName = "Heading");
+
+			PageEditorMapping.mapEditableFieldToCollectionItems(
+				field = "Title",
+				fragmentName = "Heading",
+				id = "element-text");
+
+			PageEditor.clickPublish();
+		}
+
+		task ("View the Titles are shown on the Page") {
+			ContentPagesNavigator.openViewContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			var n = "1";
+
+			for (var title : list "Web Content Title 2,Web Content Title 1") {
+				AssertVisible(
+					key_colPosition = "1",
+					key_entryContent = "${title}",
+					key_rowPosition = "${n}",
+					locator1 = "Fragment#CONTENT_DISPLAY_FRAGMENT_CONTENT");
+
+				var n = ${n} + 1;
+			}
+		}
+
+		task ("Select the Segment on Simulation Panel") {
+			Click(locator1 = "ControlMenu#SIMULATION");
+
+			Check.checkNotVisible(
+				checkboxName = "Segment Name",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+		}
+
+		task ("View the Title with segment is shown on the Page") {
```

Comment:
View only the web content with tag shown in Simulation modal when select segment